app/models/decorators/mvi_user_decorator.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
-class Decorators::MviUserDecorator
-  MVI_SERVICE = VetsAPI::Application.config.mvi_service
+require 'mvi/service_factory'
 
+class Decorators::MviUserDecorator
   def initialize(user)
     @user = user
+    @mvi_service = MVI::ServiceFactory.get_service(mock_service: ENV['MOCK_MVI_SERVICE'])
```

Comment:
👍 adding that devops ticket now, but if missing the factory defaults it to false (e.g. if not set in prod you'd get the real one)